src/styles/card.scss

```diff
@@ -0,0 +1,106 @@
+$text-color: #616070;
+$button-text-color: #00acdc;
+
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  box-sizing: border-box;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+  padding-inline: 16px;
+  transition: all 300ms linear;
+
+  &:hover{
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 3px 0;
```

Comment:
The image dimensions are hard-coded. This may cause the image to distort if its original proportions are not the same. It's better to set only width or height and let the other dimension be automatically set by preserving the aspect ratio.